test/gtest/common/mem_buffer.cc

```diff
@@ -220,23 +228,27 @@ void mem_buffer::pattern_check(const void *buffer, size_t length, uint64_t seed)
     size_t remainder = (end - (const char*)ptr);
     if (remainder > 0) {
         ucs_assert(remainder < sizeof(*ptr));
-        uint64_t mask = UCS_MASK_SAFE(remainder * 8 * sizeof(char));
+        uint64_t mask  = UCS_MASK_SAFE(remainder * 8 * sizeof(char));
         uint64_t value = 0;
         memcpy(&value, ptr, remainder);
         if (value != (seed & mask)) {
-             UCS_TEST_ABORT("At offset " << ((const char*)ptr - (const char*)buffer) <<
-                            " (remainder " << remainder << ") : " <<
-                            "Expected: 0x" << std::hex << (seed & mask) << " " <<
-                            "Mask: 0x" << std::hex << mask << " " <<
-                            "Got: 0x" << std::hex << value << std::dec);
-         }
+            ptrdiff_t offset = UCS_PTR_BYTE_DIFF(buffer, ptr);
+            UCS_TEST_ABORT("At address "
```

Comment:
maybe add common macro/function to reduce code duplication?
`mask` will be `std::numberic_limits<uint64_t>::max()` for the checking by `seed` above